test/functionalTest/cases/0725_multiservice_ignore/service_ignore_without_multiservice.test

```diff
@@ -101,10 +102,11 @@ Date: REGEX(.*)
 =====================================================
 1
 0
+ftest
 --TEARDOWN--
```

Comment:
Yes, this is about the ftest-ftest... problem.
Will help us to detect exactly *where* the problem arises ...